week5.2-ex1.py

```diff
@@ -0,0 +1,18 @@
+
+def join(*lists, separator='-'):
+    """Join all the lists to long list with separate sign between every list"""
+    if not lists:  # if there is no list at all
+        return None
+    # join the first list and then add the separator and the rest of the lists in loop
+    new_list = [element for element in lists[0]]
+    for current_list in lists[1:]:
```

Comment:
Try to read about zip, and think of a way to reduce this for loop using https://stackoverflow.com/questions/18551458/how-to-frame-two-for-loops-in-list-comprehension-python